support/scripts/checkpatch.pl

```diff
@@ -2548,6 +2548,25 @@ sub process {
 			     "Patch subject line does not follow Unikraft scheme: '[Selector]/[Component]: [Short message]'\n" . $herecurr);
 		}
 
+# Check that the first line in the commit message is below
+# 70 characters to comply with Github UI. This helps
+# avoid commit message summaries placed at the beginning
+# to overflow into the second line when viewing through
+# the Github UI.
+		if ($in_header_lines &&
+		    $line =~ /^Subject:.*$/i) {
+			my $commit_summary_line0 = $1 if
+				$lines[$linenr - 1] =~ /^Subject: \[.*\] (.*)$/;
+			my $commit_summary_line1 = $lines[$linenr];
+			my $commit_summary = $commit_summary_line0 .
+					     $commit_summary_line1;
+			if (length($commit_summary) > 70) {
+				WARN("COMMIT_LOG_LONG_LINE",
```

Comment:
Should we maybe have this as an error instead of warning? The GitHub interface messes up the commits pretty bad, so I would enforce this as much as possible.

Reply:
IMO as we should not strictly enforce it - ie block a PR - given that there could be legitimate reasons for the subject line to exceed that limit, it makes sense if it is a warning. On the other hand we don't really differentiate much between warnings and errors anyway (or do we)?

Reply:
Agree with @michpappas . Keep this as a warning.